cocos/core/math/quat.ts

```diff
@@ -349,6 +370,7 @@ export class Quat extends ValueType {
     }
 
     /**
+     * @en Conjugating a quaternion, and the unit quaternion equivalent to inversion, but more efficient
```

Comment:
Conjugating a quaternion, it's equivalent to the inverse of the unit quaternion, but more efficient